openwisp_controller/pki/api/urls.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.urls import path
+
+from . import views as api_views
+
+app_name = 'openwisp_controller'
+
+
+def get_pki_api_urls(api_views):
+    """
+    returns:: all the API urls of the PKI app
+    """
+    if getattr(settings, 'OPENWISP_CONTROLLER_PKI_API', True):
+        return [
+            path('ca/', api_views.ca_list, name='ca_list'),
+            path('ca/<str:pk>/', api_views.ca_detail, name='ca_detail'),
+            path('ca/<str:pk>/crl', api_views.crl_download, name='crl_download'),
+            path('cert/', api_views.cert_list, name='cert_list'),
```

Comment:
| SN | /api/v1/pki/cert/                             | Status |                                Details                                |
|:--:|-----------------------------------------------|:------:|:---------------------------------------------------------------------:|
|  1 | [SU][O1A][O1O] Can view only appropriate ca   |  Pass  |                                                                       |
|  2 | [SU][O1A][O1O] Can create only appropriate ca |  Fail  | When trying to import certificate, I need to provide validity period. |
|  3 | [SU][O1A] All errors reported together        |  Pass  |                                                                       |

Reply:
> When trying to import certificate, I need to provide validity period.

@atb00ker By this do you mean `validity_start` & `validity_end`, this fields are not required fields, and even if the data get''s sent in a POST request, it gets replaced with the `cert` data from which it is imported.

Reply:
@ManishShah120 Are you sure they are not required? It wouldn't let me submit on firefox without these fields!

Reply:
Yes, I am able to submit it, I also tested it in JSON format form.

Reply:
I confirm that I have been able to import as expected and without issues on Chrome,
while on Firefox as @atb00ker noted, the validity start and end fields are somehow prefilled and hence submitting the form fails with a validation error unless the prefilled value is edited manually.

Not a big deal, because this will happen only when using it from the browsable API. @ManishShah120 can you double check on Firefox and see if you can force the default value to be empty there too?